### PR TITLE
Implement Results Downloading Functionality

### PR DESCRIPTION
**Pull request by bzzbbz**
**tl;dr: Survey results can now be downloaded as an xlsx workbook in the MAnage Responses panel.**

### More details

- Each row of the excel spreadsheet contains the Tasker's address, the time of completion, and responses to each question.
- Only approved responses gets into the excel spreadsheet.

### Next steps (very minor things):
- @christine-sun @jennsun @nicholaspad The placement of the Download button is a bit odd right now, need suggestions on where else to put it.
- @nicholaspad Very minor thing, it'll be great if you can add a hover function over the CTA buttons that shows a little bubble of text when mouse hovers over them. I mainly want to add "Only approved tasks are entered into the Excel file" when users hover over the "Download Results" button.
- @bzzbbz Batch approve/reject (if time allows tomorrow)

### CODE REVIEW

**Comment by jennsun** [comment]
I think the placement of the download button is fine, but perhaps change it to the secondary button to make it distinct from the <- My Tasks?

File: pages/requester/manage-task/[taskid].tsx [hunk 1]
```diff
@@ -185,25 +219,44 @@ export default function ManageResponses() {
...
+        />
+      </Box>
       <ApprovalsTable data={data} extraColumns={extraColumns} />
     </>
   );
 }
 
 const approvalStatusMap = {
-  1: "Not reviewed",
+  1: "Not Reviewed",
```
Comment (jennsun) [inline pages/requester/manage-task/[taskid].tsx:249]: Let's keep these consistent across the board with the other status map:
  0: "In progress",
  1: "Pending approval",
  2: "Ready to Withdraw",
  3: "Abandoned",
  4: "Paid",
  5: "Rejected",

**Review by jennsun — CHANGES_REQUESTED** [review]
Looks good! Had a comment about the Download button design and question about changing the status map labels

**Review by jennsun — APPROVED** [review]
